code/game/objects/structures/dropship_ammo.dm

```diff
@@ -28,6 +28,16 @@
 	///Type of ammo
 	var/ammo_type
 
+	///Explosion ratings, for explosive weapons
+	var/devastating_explosion_range = 0 //HE OB uses this
+	var/heavy_explosion_range = 0 //Mortar splash damage uses this
+	var/light_explosion_range = 0 //HEDP uses this
+	///Fire radius, for incendiary weapons
```

Comment:
A) autodoc
B) I dont know what x uses this is supposed to men please make useful comments or remove them